scripts/install-chef-suse.sh

```diff
@@ -1166,6 +1154,18 @@ for s in xinetd dhcpd apache2 ; do
     fi
 done
 
+# activate provisioner repos
+if ! is_ses; then
+    for i in SLES12-SP1-Pool SLES12-SP1-Updates SUSE-OpenStack-Cloud-6-Pool SUSE-OpenStack-Cloud-6-Updates SLES12-SP1-HA-Pool SLES12-SP1-HA-Updates; do
```

Comment:
In https://github.com/crowbar/crowbar-core/pull/102, I added a `activate_all` API that you can use instead of iterating over a list of repos.